### PR TITLE
Doc/#18 updated link to impala product page

### LINKED ISSUE #18: Fix broken link in user guide
```
FILE: ./doc/user_guide/impala_user_guide.md
[✖] https://www.cloudera.com/products/open-source/apache-hadoop/impala.html → Status: 404

7 links checked.
```

### PR DESCRIPTION
**Pull request by ckunki**
Closes #18

### CODE REVIEW

File: .settings/org.eclipse.core.resources.prefs
Comment (kaklakariada) [inline .settings/org.eclipse.core.resources.prefs:1]: Delete this file and ignore it

File: .settings/org.eclipse.m2e.core.prefs
Comment (kaklakariada) [inline .settings/org.eclipse.m2e.core.prefs:1]: Delete & ignore this

File: doc/changes/changes_2.0.1.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Virtual Schema for Impala 2.0.1, released 2024-??-??
+
+Code name:
+
+## Summary
+
+## Documentation
+
+* #18: Updated link to claudera impala page and dependencies
```
Comment (kaklakariada) [inline doc/changes/changes_2.0.1.md:9]:
```suggestion
* #18: Updated link to Cloudera Impala page and dependencies
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: .settings/org.eclipse.core.resources.prefs
Comment (ckunki) [reply .settings/org.eclipse.core.resources.prefs:1]: See next push

File: .settings/org.eclipse.m2e.core.prefs
Comment (ckunki) [reply .settings/org.eclipse.m2e.core.prefs:1]: See next push

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**